app/controllers/register-service.controller.js

```diff
@@ -150,7 +150,7 @@ async function showOtpVerify (req, res, next) {
   }
 }
 
-async function createPopulatedService (req, res, next) {
+async function submitOtpCode (req, res, next) {
```

Comment:
This is `submitOtpVerify` in `register-user.controller` - maybe it would be worth harmonising the names?

Reply:
Good point. I would agree if these controllers were sticking around, but our aim is to kill them off in the near future in favour of our new controllers for the combined journey so this inconsistency won't affect people for very long. So I won't do it this time if that's ok, just so I don't have to rebuild the PR.

Reply:
👍